tests/models/self_supervised/test_models.py

```diff
@@ -18,13 +20,14 @@ def test_cpcv2(tmpdir):
     datamodule.val_transforms = CPCEvalTransformsCIFAR10()
 
     model = CPCV2(encoder='resnet18', data_dir=tmpdir, batch_size=2, online_ft=True, datamodule=datamodule)
-    trainer = pl.Trainer(fast_dev_run=True, max_epochs=1, default_root_dir=tmpdir)
+    trainer = pl.Trainer(fast_dev_run=True, max_epochs=1, default_root_dir=tmpdir, gpus=1)
```

Comment:
these tests are not running on GPU
```suggestion
    trainer = pl.Trainer(fast_dev_run=True, max_epochs=1, default_root_dir=tmpdir)
```